proofs.md

```diff
@@ -69,15 +70,15 @@ Seal
 VerifySeal
  (
   // request represents a request to verify the output of a Seal() operation.
-  commD     [32]byte,                       // returned from Seal
-  commR     [32]byte,                       // returned from Seal [will be removed in future iteration] 
-  commRStar [32]byte,                       // returned from Seal
-  proof     []byte,                         // returned from Seal
-  proverID  [31]byte,                       // uniquely identifies miner
-  ticket                                    // identifies ticket provided when sealing
-  sectorID  [31]byte,                       // uniquely identifies sector
+  commD       [32]byte, // returned from Seal
+  commR       [32]byte, // returned from Seal [will be removed in future iteration]
+  commRStar   [32]byte, // returned from Seal
+  proof       []byte,   // returned from Seal
+  proverID    [31]byte, // uniquely identifies miner
+  roundNumber uint64,   // round (block height) to which a miner committed when sealing began
```

Comment:
in the interface to the actual function i think the ticket is correct. We just don't want to submit the entire ticket to the chain in a message (let the chain look up the ticket and pass it to this method)

Reply:
Yes — both sealing and verification will need the actual ticket bytes, not the round number.

Reply:
> both sealing and verification will need the actual ticket bytes, not the round number.

Ah, yeah - that makes more sense.